java/build.gradle

```diff
@@ -8,6 +8,20 @@ sourceCompatibility = targetCompatibility = 8
 archivesBaseName = 'FlexVer'
 version = '1.0.2'
 
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.2'
+	implementation 'org.junit.jupiter:junit-jupiter-params:5.9.2'
```

Comment:
This will pull in this dep transitively on Maven! Needs to be `compileOnly` or `testImplementation`.